client/knowledge_base_search/src/containers/SearchContainer/SearchContainer.jsx

```diff
@@ -157,6 +157,7 @@ class SearchContainer extends Component {
             <FeatureSelect
               onFeatureSelect={this.props.onFeatureSelect}
               selectedFeature={this.props.selectedFeature}
+              isFetchingResults={this.props.isFetchingResults}
```

Comment:
minor nitpick. The other file does deconstruction of the prop vars. Maybe do that here too?

```js
const { onFeatureSelect, selectedFeature, isFetchingResults } = this.props;
```